conda_build/conda_interface.py

```diff
@@ -72,8 +64,36 @@
     walk_prefix,
     win_path_to_unix,
 )
+from conda.exports import display_actions as _display_actions
+from conda.exports import execute_actions as _execute_actions
+from conda.exports import execute_plan as _execute_plan
+from conda.exports import get_index as _get_index
+from conda.exports import install_actions as _install_actions
+from conda.exports import linked as _linked
+from conda.exports import linked_data as _linked_data
+from conda.exports import package_cache as _package_cache
 from conda.models.channel import get_conda_build_local_url  # noqa: F401
-from conda.models.dist import Dist  # noqa: F401
+from conda.models.dist import Dist as _Dist
+from conda.models.records import PackageRecord
+
+from .deprecations import deprecated
+
+deprecated.constant(
+    "3.28.0",
+    "24.1.0",
+    "IndexRecord",
+    PackageRecord,
+    addendum="Use `conda.models.records.PackageRecord` instead.",
+)
```

Comment:
Whoops, accidentally added back the `IndexRecord`:

```suggestion

from .deprecations import deprecated

```